src/lib.rs

```diff
@@ -28,6 +28,7 @@
 //! assert_eq!(items, [(2, 20), (1, 10), (3, 30)]);
 //! ```
 
+#![forbid(missing_docs)]
```

Comment:
didn't want to hold you up, but there's not much difference between `deny` and `forbid` here is there?